src/_listDefinitions.scss

```diff
@@ -40,6 +40,27 @@ $fd-list-navigation-navigator-bar-right: inset -0.1875rem 0 0 0 var(--sapSelecte
 $fd-list-navigation-navigator-bar-left: inset 0.1875rem 0 0 0 var(--sapSelectedColor);
 $fd-list-navigation-indicator-width: 2.75rem !default;
 
+$fd-list-optional-text-font-family: var(--sapUiFontFamily);
```

Comment:
Is this the right variable? I don't believe `sapUiFontFamily` is something that is a part of @sap-theming repo

https://github.com/SAP/theming-base-content/blob/master/content/Base/baseLib/sap_fiori_3/css_variables.css

Additionally, things like fontFamily should be set by the reset and should be unecessary

Reply:
updated sapUiFontFamily, where ever usage of fd-reset is not causing issues updated it as well.